src/streamlink/plugins/twitch.py

```diff
@@ -565,19 +565,15 @@ async def acquire_client_integrity_token(client: CDPClient):
                     await client_session.loaded(frame_id)
                     return await client_session.evaluate(js_get_integrity_token, timeout=eval_timeout)
 
-        def handle_error(exc_grp: BaseExceptionGroup) -> None:
-            for err in exc_grp.exceptions:
-                log.error(f"{type(err).__name__}: {err}")
-
-        with catch({  # type: ignore[dict-item]  # bug in exceptiongroup==1.2.0
-            Exception: handle_error,  # type: ignore[dict-item]  # bug in exceptiongroup==1.2.0
-        }):
+        try:
             client_integrity = CDPClient.launch(
                 session,
                 acquire_client_integrity_token,
                 # headless mode gets detected by Twitch, so we have to disable it regardless the user config
                 headless=False,
             )
+        except BaseExceptionGroup:
```

Comment:
On recent python version, if there is a single exception, will this still catch it if the BaseExceptionGroup is unwraped ?
I'm not sure if this can really happen though as tests are passing.

Reply:
`trio.run()` is called with `strict_exception_groups=True`, so only exception groups can be raised from `trio.Nursery` contexts. That was the whole reason why trio's 0.25 was a breaking change for us and why we now have to set this argument explicitly on older versions of trio.

There are a couple of cases though where we actually want to catch and log non-group exceptions and then continue running the plugin code without retrieving the client-integrity token, namely `resolve_executable()`, `find_free_port_ipv4()`, `tempfile.TemporaryDirectory()`, etc. Let me fix this real quick...

Reply:
Indeed I forgot that `strict_exception_groups=True` option, thanks for the explanation.